### DISCUSSION TITLE
PCE voltage / side switch

### DISCUSSION THREAD

**Discussion by Ryurron** [sanni/cartreader#464] [Need help]
Hello
What is the best voltage and side switch setting for dumping PCE?
Dumping is successful in any setting.
We have tried it on a few cards.
We want to know what is the best for dumping PCE.
Thank you.

**Comment by sanni** [comment]
![](https://dl.dropboxusercontent.com/s/5np6jvcv7x6ilvl/flash_jumper.jpg?dl=1)

**Reply by Ryurron** [reply]
Thank you ;)
